interface/client/src/ui/UiModule.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import * as store from '../store';
+import {connect} from 'react-redux';
+import {ScAddr} from '@ostis/sc-core';
+import {useEffect} from 'react';
+
+export interface KnowledgeBaseProps {
+    services: store.serv.Services;
+}
+
+function mapStateToProps(state: store.Store): KnowledgeBaseProps {
+    return {
+        services: state.services,
+
+    };
+}
+
+export const UiModuleImpl: React.FC<KnowledgeBaseProps> = (props: KnowledgeBaseProps) => {
+
+    const {services} = props;
+
+    const ui = async () => {
+        const addrs: ScAddr[] = await services.sc.utils.CallAnswerAgent();
+        const questionNode: ScAddr = addrs[0];
+
+        const actionNode: ScAddr = await services.sc.utils.SearchActionNode(questionNode);
```

Comment:
Could you please explain the difference between `questionNode` and `actionNode`?? From our context it's the same